guide/src/python_from_rust.md

```diff
@@ -236,6 +236,75 @@ def leaky_relu(x, slope=0.01):
 # }
 ```
 
+### Want to embed Python in Rust with exposed modules?
+
+Internally Python maintains the `sys.modules` dict.
+An import in Python is simply a lookup into this dict.
+
+It is necessary to insert the new module into the dict
+before it is exposed to Python. There are two ways of
+doing this.
+
+   * manually inserting into `sys.modules`
+   * using [`append_to_inittab`]({{*PYO3_DOCS_URL}}/pyo3/macro.append_to_inittab.html) macro
```

Comment:
```suggestion
   * declaring the module using the `pymodule` macro and using the [`append_to_inittab`]({{*PYO3_DOCS_URL}}/pyo3/macro.append_to_inittab.html) macro
```

Reply:
I personally find this verbose using [`append_to_inittab`] communicates the point...

Reply:
The point is that they are two very different things, which have different use cases. You can't use `append_to_inittab` with `PyModule::new` created modules and vice versa.